web/frontend/angular/src/main.ts

```diff
@@ -1,6 +1,8 @@
 import {enableProdMode} from "@angular/core"
 import {platformBrowserDynamic} from "@angular/platform-browser-dynamic"
 
+import "bootstrap"
```

Comment:
I don't think this is necessary. According to documentation, importing and using `NgbModule` is sufficient.